cvprac/cvp_api.py

```diff
@@ -2619,3 +2619,205 @@ def deploy_device(self, device, container, configlets=None,
         if create_task:
             return self._save_topology_v2([])
         return None
+
+    # pylint: disable=invalid-name
+    # Must be discussed with issue aristanetworks/cvprac #94
+    def get_net_element_info_by_device_id(self, device_id):
```

Comment:
Can this be replaced by existing function get_device_by_mac or get_device_by_name?

Reply:
Tested with success using cv_facts